flink-python/docs/generate-docs.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+TARGET_PATH=
+# parse_opts
+USAGE="
+usage: $0 [options]
+-h          print this help message and exit
+-t          list all checks supported.
+"
+while getopts "ht:" arg; do
+    case "$arg" in
+        h)
+            printf "%s\\n" "$USAGE"
+            exit 2
+            ;;
+        t)
+            TARGET_PATH=$OPTARG
+            ;;
+        ?)
+            printf "ERROR: did not recognize option '%s', please try -h\\n" "$1"
+            exit 1
+            ;;
+    esac
+done
+
+LINTPYTHON_PATH="../dev/lint-python.sh"
```

Comment:
The means that we must execute this script under directory doc.
What about changed as follows:
CURRENT_DIR="$(cd "$( dirname "$0" )" && pwd)"
LINTPYTHON_PATH="$CURRENT_DIR/../dev/lint-python.sh"